docs/Telemetry.md

```diff
@@ -0,0 +1,53 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+
+## Telemetry
+
+Accessibility Insights for Windows uses telemetry to better understand what features are most helpful to users, as well as to help identify potential issues that users are experiencing.
```

Comment:
I'm assuming we will eventually need to change references in this file from Accessibility Insights to Axe Windows?

Reply:
Yes, this file needs to be rewritten for Axe.Windows. It is on my list of things to do.

Reply:
Yes. We need to rewrite this file for Axe.Windows.